dts/bindings/can/renesas,ra-canfd.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas RA CANFD controller
+
+compatible: "renesas,ra-canfd"
+
+include: [can-fd-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  interrupts:
+    required: true
+
+  clocks:
+    required: true
+
+  channel:
+    type: int
+    required: true
+
+  rx-max-filters:
```

Comment:
Renesas specific property should have renesas, prefix.

Reply:
Only when used on generic compatibles. Here, the compatible is already Renesas-specific, so the whole namespace is Renesas.